_data/changelogs/component-button.yml

```
@@ -2,6 +2,14 @@ title: Button
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Improved the vertical alignment of `usa-icon` elements inside of buttons.
+    summaryAdditional: Spacing can be adjusted using the `$theme-button-icon-gap` setting.
```

Wondering if the addition of this setting should be its own separate entry. Either way, I think it would be helpful if we clarified what "spacing" means. Maybe something like this?
```suggestion
    summaryAdditional: Also added the `$theme-button-icon-gap` setting, which adjusts the size of the gap between the button's text and icon.
```

---

I like the idea of separating the visual from the component setting. I've added a new entry for `$theme-button-icon-gap` [de4cec1].